hack/update-schemas.sh

```diff
@@ -18,20 +18,34 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
-# Create a backup for every linked CRD.
-links=$(find "$(dirname "$0")/../config/core/300-resources" -type l)
-for link in $links; do
-  cp "$link" "$link.bkp"
-done
-
-# Make sure you install the patched version of controller-gen from
-# https://github.com/markusthoemmes/controller-tools/tree/knative-specific
+# Install patched schemagen into a temporary directory.
 #
 # We need a patched version because
 # 1. There's a bug that makes our URL types unusable
 #    see https://github.com/kubernetes-sigs/controller-tools/issues/560
 # 2. We need specialized logic to filter down the surface of PodSpec we allow in Knative.
 #    The respective config for this is in `schemapatch-config.yaml`
+export GOBIN
+GOBIN=$(mktemp -d)
```

Comment:
codegen lib actually defines GOBIN. So you may not even need this - or set it when the var is empty for resiliency 


I discovered this when here https://github.com/knative-sandbox/net-istio/blob/main/hack/update-codegen.sh#L25-L27

Reply:
Does that make any diff here? I'm not even importing the codegen lib as I'm hacking around anyway :see_no_evil:

Reply:
you're invoking update-schemas from update-codegen which does import the codegen lib

ie.
```
source $(dirname $0)/../vendor/knative.dev/hack/codegen-library.sh
```

It might not be an issue 🤷